libfwupd/fwupd-client.c

```diff
@@ -818,10 +819,18 @@ fwupd_client_connect_async(FwupdClient *self,
 		return;
 	}
 
-	/* use peer-to-peer only if the env variable is set */
+	/* convert from filename to address, if required */
```

Comment:
I think you want to do something special here for
`#ifdef _WIN32` actually to make fwupd clients work more "automagically".  Maybe look if it's NULL then hardcode to `tcp:...`?

Reply:
That's a good idea. what port do we use tho? we have to set the same port on the client and the server, and I don't think there's any way to "discover" what port the daemon chose if you leave it to choose something unused.

Reply:
How about 1341?  It seems like https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?=&skey=2&page=22 it's only used for Qubes today.

Reply:
Good idea, fixed.